src/core/etl/src/Flow/ETL/Row/Reference/Expression/Sanitize.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Row\Reference\Expression;
+
+use Flow\ETL\Row;
+use Flow\ETL\Row\Reference\Expression;
+
+final class Sanitize implements Expression
+{
+    public function __construct(
+        private readonly Expression $ref,
+        private readonly string $placeholder,
+        private readonly int $charactersLeft
+    ) {
+    }
+
+    public function eval(Row $row) : ?string
+    {
+        /** @var mixed $val */
+        $val = $this->ref->eval($row);
+
+        if (!\is_string($val)) {
+            return null;
+        }
+
+        $size = \strlen($val);
```

Comment:
I think we should use here mb_strlen as input may contain multibyte characters, such as UTF-8

Reply:
Done.